packages/block-library/src/post-template/edit.js

```diff
@@ -152,20 +152,30 @@ export default function PostTemplateEdit( {
 					per_page: -1,
 					context: 'view',
 				} );
-				// We have to build the tax query for the REST API and use as
-				// keys the taxonomies `rest_base` with the `term ids` as values.
-				const builtTaxQuery = Object.entries( taxQuery ).reduce(
-					( accumulator, [ taxonomySlug, terms ] ) => {
-						const taxonomy = taxonomies?.find(
-							( { slug } ) => slug === taxonomySlug
-						);
-						if ( taxonomy?.rest_base ) {
-							accumulator[ taxonomy?.rest_base ] = terms;
-						}
-						return accumulator;
-					},
-					{}
-				);
+				// Shared utility to build REST API parameters from taxonomy terms.
```

Comment:
```suggestion
				// Build REST API parameters from taxonomy terms, e.g.
				// `category`, `tags_exclude`.
```

(Just from reading the function signature, none of this is very clear, IMO. It's not a big deal because it's a throwaway helper, but I thought I'd share my perspective as reader.)

Edit: hm, I see now that the same function is duplicated in other files. That's fine, but calling it "shared utility" may confuse more than not.